ui/package.json

```diff
@@ -20,5 +20,8 @@
     "devDependencies": {
         "rimraf": "5.0.7",
         "shelljs": "0.8.5"
+    },
+    "dependencies": {
+        "@apicurio/apicurio-registry-client": "file:../typescript-sdk"
```

Comment:
this uses `npm link` ... not sure if we can improve it anyhow ...

Reply:
ok - i'll see what I can do on this